static/styles/portico/integrations_dev_panel.css

```diff
@@ -10,6 +10,34 @@
     padding: 20px 10px 15px;
     background-color: hsla(0, 0%, 100%, 0.48);
     box-shadow: 0 0 40px hsla(0, 0%, 0%, 0.06);
+
+    textarea {
+        color: hsl(0, 0%, 33%);
+        background-color: hsl(0, 0%, 100%);
+        border-radius: 4px;
+        vertical-align: middle;
+        border: 1px solid hsl(0, 0%, 80%);
+        padding: 4px 6px;
+        margin-bottom: 10px;
+
+        font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
```

Comment:
I don't think we care about the visual design of this page beyond it being vaguely usable -- so I would prefer that we include a much more minimal CSS that doesn't include things like a special font not used elsewhere in the app.

(But I'm fine with whatever for this page -- it's a devtool).

Reply:
For rest of the app the `font-family` property is set in `zulip.css`, but since CSS in `zulip.css` are not applied to portico pages, so I have set it to the same value as was used by bootstrap. The other inputs, i.e. select and text input also have the same font set by bootstrap, so we can probably make font for all of them same as we use for rest of the app together.